src/appshell/qml/Preferences/AppearancePreferencesPage.qml

```diff
@@ -42,33 +42,97 @@ PreferencesPage {
     Column {
         id: content
 
+        QtObject {
+            id: prv
+
+            readonly property int firstColumnWidth: 212
+            readonly property int spacing: 24
+        }
+
         width: parent.width
         height: childrenRect.height
 
-        spacing: 24
+        spacing: prv.spacing
 
-        readonly property int firstColumnWidth: 212
+        StyledTextLabel {
+            text: highContrastEnable.checked ? qsTrc("appshell", "High Contrast Themes") : qsTrc("appshell", "Themes")
+            font: ui.theme.bodyBoldFont
+        }
 
-        ThemesSection {
-            width: parent.width
+        CheckBox {
+            id: highContrastEnable
+            width: 200
+            checked: appearanceModel.enableHighContrastChecked()
+
+            text: qsTrc("appshell", "Enable High-Contrast")
```

Comment:
Sentence case, not Title Case

Reply:
(But yet, our policy there just changed recently)

Reply:
Converted to `"Enable high-contrast"`
Fixed.